src/Provider/AbstractProvider.php

```diff
@@ -445,7 +445,9 @@ protected function getAccessTokenBody(array $params)
      */
     protected function getAccessTokenOptions(array $params)
     {
-        $options = [];
+        $options = [
+          'headers' => $this->getAccessTokenRequestHeaders($params),
+        ];
 
         if ($this->getAccessTokenMethod() === self::METHOD_POST) {
             $options['body'] = $this->getAccessTokenBody($params);
```

Comment:
You wouldn't use the trait for your provider if you're using GET. It'll just be the default empty headers.